### PR TITLE
Prior plotting

### PR DESCRIPTION
**Pull request by ManInFez**
Using scipy stats module to plot theoretical priors, not all are complete, as the sampling is not always folloing the theoretical mathematical function it says it does.

resolves https://github.com/equinor/ert/issues/772

### CODE REVIEW

File: ertviz/controllers/ensemble_selector_controller.py [hunk 1]
```diff
@@ -78,11 +78,7 @@ def update_ensemble_selector(query, selected_ensembles, elements):
             ensemble_dict = get_ensembles(project_id=parent.project_identifier)
             for ensemble_schema in ensemble_dict:
                 ensemble_id = ensemble_schema["id"]
-                if ensemble_id not in parent.ensembles:
-                    parent.ensembles[ensemble_id] = EnsembleModel(
-                        ensemble_id=int(ensemble_id),
-                        project_id=parent.project_identifier,
-                    )
+                get_ensemble(parent, ensemble_id)
```
Comment (xjules) [inline ertviz/controllers/ensemble_selector_controller.py:81]: Shouldn't this provide a handle to ensemble?

Comment (ManInFez) [reply ertviz/controllers/ensemble_selector_controller.py:81]: maybe the name is wrong, as it not only returns the ensemble, but loads it...

Comment (ManInFez) [reply ertviz/controllers/ensemble_selector_controller.py]: renamed function to load ensemble :)

File: ertviz/controllers/multi_parameter_controller.py [hunk 2]
```diff
@@ -65,19 +58,25 @@ def _update_histogram(parameter, hist_check_values, selected_ensembles):
...
-                ][parameter].data_df()
-                colors[parent.ensembles[ensemble_id]._name] = color["color"]
+            ensemble = load_ensemble(parent, ensemble_id)
+            if parameter in ensemble.parameters:
+                parameter_model = ensemble.parameters[parameter]
+                data[ensemble._name] = parameter_model.data_df()
+                colors[ensemble._name] = color["color"]
+
+                if parameter_model.priors and "prior" in hist_check_values:
+                    priors[ensemble._name] = (parameter_model.priors, color)
```
Comment (xjules) [inline ertviz/controllers/multi_parameter_controller.py:70]: Shouldn't we use ids instead as we can have 2 ensembles with the same name?

Comment (ManInFez) [reply ertviz/controllers/multi_parameter_controller.py:70]: yes wasnt sure if you had made that change in your pr

Comment (xjules) [reply ertviz/controllers/multi_parameter_controller.py:70]: If you mean histogram then now, but can update it

File: ertviz/controllers/multi_parameter_controller.py [hunk 3]
```diff
@@ -109,3 +108,29 @@ def _set_parameter_from_btn(_prev_click, _next_click, parameter_options, paramet
...
+        Output(parent.uuid("hist-check"), "options"),
+        [
+            Input(parent.uuid("parameter-selector"), "value"),
+        ],
+        [
+            State(parent.uuid("hist-check"), "options"),
+            State(parent.uuid("ensemble-selection-store"), "data"),
+        ],
+    )
+    def _set_parameter_from_btn(parameter, plotting_options, selected_ensembles):
```
Comment (xjules) [inline ertviz/controllers/multi_parameter_controller.py:122]: Not sure, if this's necessary. Or if can you just make it disabled instead?

File: ertviz/controllers/multi_parameter_controller.py (hunk 2 above)
Comment (xjules) [inline ertviz/controllers/multi_parameter_controller.py:70]: Hm, do you think that prior can change over different ensembles for the same parameter?

**Review by xjules — APPROVED** [review]
Tested and approved :) Bra job!

**Pull request merged**